### PR TITLE
Add gulp-sass-lint, update gitignore

### PR DESCRIPTION
**Pull request by katiehartrick**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,10 +6,6 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "browserslist": [
-    "last 1 version",
-    "> 1%"
-  ],
```
Comment (sergioloporto) [inline package.json:12]: Hello, why this change?

Comment (katiehartrick) [reply package.json:12]: Moved to  seperate `.browserslistrc` file

**Comment by katiehartrick** [comment]
upgraded to dart-sass and stylelint
